kernelci/db/kernelci_api.py

```diff
@@ -24,6 +25,14 @@
 from kernelci.db import Database
 
 
+class PendingNodeStates(enum.Enum):
+    """Enumeration for pending node states"""
```

Comment:
Maybe it's worth mentioning that this is a temporary solution to be replaced with an API call when the versioning is in place.

Reply:
Updated the docstring.